src/tester.rs

```diff
@@ -13,6 +15,21 @@ pub struct QuickCheck<G> {
     gen: G,
 }
 
+fn qc_tests() -> usize {
+    match env::var("QC_TESTS") {
+        Ok(val) => val.parse().expect("QC_TESTS value could not converted to number"),
```

Comment:
If the parse fails, we *definitely* shouldn't panic here. I think we should just return the default value if you can't parse a number. (Alternatively, we could make `QuickCheck::new` return a result, but I think just returning the default on a parse failure is fine.)

Similarly for `qc_max_tests`.